### DISCUSSION TITLE
No hardware keys detected

### DISCUSSION THREAD

**Discussion by alexbalkan** [keepassxreboot/keepassxc#8393] [Q&A] [answered]
my computer and yubikey manager sees my Yubikey 5 nano usb. But keepassxc 2.6.6 doesn't see the key. I hoped that in the new version the key would be supported. In the new version, the miracle did not happen. the problem with detecting the yubikey remained

**Comment by droidmonkey** [comment, marked as answer]
If you are on Linux and Ubuntu you need to install the yubikey udev rules. If you are using the snap you need to enable raw usb permissions.
